tests/integration/components/paper/radio-group-test.js

```diff
@@ -0,0 +1,76 @@
+import { module, test }           from 'qunit';
+import { render, findAll, click } from '@ember/test-helpers';
+import hbs                        from 'htmlbars-inline-precompile';
+import { setupRenderingTest }     from 'ember-qunit';
+
+module('Integration | Component | {{paper/radio-group}}', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('shows options', async function(assert) {
+    this.set('options', ['Option 1', 'Option 2']);
+    this.set('selection', null);
+
+    await render(hbs`{{paper/radio-group value=selection options=options}}`);
+
+    let $options = findAll('md-radio-button');
+
+    assert.equal($options.length, 2, 'shows all options');
+    assert.equal($options[0].innerText.trim(), 'Option 1', 'shows option 1 value');
```

Comment:
I thought @Haz901 has changed `innerText` to `textContent` in all places. Is that right?